src/request.spec.ts

```diff
@@ -396,10 +396,14 @@ describe('Request', () => {
   });
 
   it('returns response of a successful GET request, with a proxy url', (done) => {
+    let outcome = false;
```

Comment:
It would be clearer to name the variable after what it represents, eg. `proxyConfigured`